.github/workflows/publish.yml

```diff
@@ -0,0 +1,43 @@
+name: Publish
+
+on:
+  push:
+    branches:
+        - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y python3-venv git
+    - uses: actions/checkout@v2
+    - name: Set up doc directory
+      run: |
+        mkdir $HOME/output
+        touch $HOME/output/.nojekyll
+    - name: Build latest
+      run: |
+        rm -rf _work/venv
+        make vhtml
+        mv _build/html/* $HOME/output/
```

Comment:
I guess this move operation could be faster if it was
```
make vhtml
touch _build/html/.nojekyll
mv _build/html $HOME/output
```

Then the step "Set up doc directory" also could be removed.

Reply:
I copied these from pmem-csi. In the near future, this workflow needs to support builds for the release branch(es) and master  so there will be subdirectories under `$HOME/output` and a run stage for each branch...